pkgs/applications/networking/browsers/vivaldi/ffmpeg-codecs.nix

```diff
@@ -0,0 +1,51 @@
+{ stdenv, fetchurl, fetchpatch
+, dbus_glib, gtk2, gtk3, libexif, libpulseaudio, libXScrnSaver, ninja, nss
+, pciutils, pkgconfig, python2, xdg_utils
+}:
+
+stdenv.mkDerivation rec {
+  name = "${product}-${version}";
+  product = "vivaldi-ffmpeg-codecs";
+  version = "59.0.3071.104";
```

Comment:
I just noticed that the AUR package includes a check for the Chromium version and added that to the package. This also made me notice that the currently packaged Vivaldi version isn't the latest. Thanks for the feedback!